bootstrap_salt/config.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pkgutil
+
+import bootstrap_cfn.config
+
+from fabric.api import env
+
+import yaml
+
+
+class MyConfigParser(bootstrap_cfn.config.ConfigParser):
+
+    def base_template(self):
+        ret = super(MyConfigParser, self).base_template()
+        ret.add_mapping('KMS', {'salt': {'key': env.kms_key_id}})
```

Comment:
Can we make this not rely on the fabric env but instead read from instance variables/properties (i.e. `self.kms_key_id` or `self.data['kms_key_id']`) It should make this easier to test too.

Speaking of: can haz tests?